src/app/plan/duck/sagas.ts

```diff
@@ -120,9 +120,7 @@ function* planPatchClose(planValues) {
       closedPlanSpecObj
     );
     yield put(PlanActions.updatePlanList(patchPlanResponse.data));
-    yield put(PlanActions.planUpdateSuccess());
```

Comment:
Is this intended?

Reply:
Yep - there is no more plan update action.